tor2web/utils/lists.py

```diff
@@ -258,7 +267,4 @@ def handleData(self, data):
             self.clear()
 
         data = json.loads(data)
-        for relay in data['relays']:
-            for ip in relay['a']:
-                if ip != '':
-                    self.add(ip)
+        self.extend( [ ip for ip in relay['a'] for relay in data['relays'] if ip ] )
```

Comment:
This line has the wrong order.  It needs to instead be:

self.extend( [ ip for relay in data['relays'] for ip in relay['a'] if ip ] )